src/tests/test_stdlib_fixes.py

```diff
@@ -237,3 +237,60 @@ def test_encodings_deepfrozen(selenium):
     for enc in all_encodings:
         codecs.getencoder(enc)
         codecs.getdecoder(enc)
+
+
+@run_in_pyodide
+def test_zipimport_traceback(selenium):
+    import json.decoder
```

Comment:
Maybe we could add a short comment about why this test is needed. Took me some time to remember why we do this )